nailgun/entities.py

```diff
@@ -2666,6 +2666,22 @@ def disable_rhrepo(self, base_arch,
         )
         return _handle_response(response, self._server_config, synchronous)
 
+    def list_available_rhrepos(self, reposet_id):
```

Comment:
Can you rename this to `repository_sets_id_available_repositories`? I know that this is a very clunky name. But I'd like to make sure that every extra method we add to NailGun maps directly to some underlying path. I know that we already have several methods such as `enable_rhrepo` and `disable_rhrepo`. But I think that those methods should also be renamed and brought in line with a better naming scheme.